contrib/drivers/lvm/lvm.go

```diff
@@ -545,6 +626,14 @@ func (d *Driver) ListPools() ([]*model.StoragePoolSpec, error) {
 	return pols, nil
 }
 
+func (d *Driver) InitializeSnapshotConnection(opt *pb.CreateSnapshotAttachmentOpts) (*model.ConnectionInfo, error) {
+	return nil, &model.NotImplementError{S: "Method InitializeSnapshotConnection has not implement yet."}
+}
+
+func (d *Driver) TerminateSnapshotConnection(opt *pb.DeleteSnapshotAttachmentOpts) error {
+	return &model.NotImplementError{S: "Method TerminateSnapshotConnection has not implement yet."}
```

Comment:
InitializeSnapshotConnection and TerminateSnapshotConnection in LVM driver should be implemented.  It should just use the local path without the real attaching/detaching.